tooling/v8-snapshot/cache/prod-win32/snapshot-meta.cache.json

```diff
@@ -3931,5 +3936,5 @@
     "./tooling/v8-snapshot/cache/prod-win32/snapshot-entry.js"
   ],
   "deferredHashFile": "yarn.lock",
-  "deferredHash": "5a515f98fe67a8a56a035563ff9c8b9e4a9edc4d035907c3fa5fef1bb60f1dfc"
```

Comment:
I've been seeing merge conflicts in these snapshot caches fairly frequently recently, as we move towards 12.0.0 release - I know you were looking at having them auto-update in develop, so we wouldn't need to do it manually. I take it that's not quite ready yet?

Reply:
Yeah not ready yet unfortunately. It'll be one of my next tasks.